src/resolver/resolve_path.zig

```diff
@@ -1728,72 +1724,6 @@ test "longestCommonPath" {
     _ = t.expect("/app/public/", longestCommonPath(more[0..2]), @src());
 }
 
-pub fn basename(path: []const u8) []const u8 {
-    if (path.len == 0)
-        return &[_]u8{};
-
-    var end_index: usize = path.len - 1;
-    while (isSepAny(path[end_index])) {
-        if (end_index == 0)
-            return &[_]u8{};
-        end_index -= 1;
-    }
-    var start_index: usize = end_index;
-    end_index += 1;
-    while (!isSepAny(path[start_index])) {
-        if (start_index == 0)
-            return path[0..end_index];
-        start_index -= 1;
-    }
-
-    return path[start_index + 1 .. end_index];
-}
```

Comment:
☝️ Removed `basename` generic from `resolve_path` in favor of `std.fs.path.basename`.